api/v1beta1/flinkcluster_types.go

```diff
@@ -54,6 +54,7 @@ const (
 	AccessScopeCluster  = "Cluster"
 	AccessScopeVPC      = "VPC"
 	AccessScopeExternal = "External"
+	AccessScopeNodePort = "NodePort"
```

Comment:
What about Loadbalancer?

Reply:
Do you mean support LoadBalancer?

Reply:
It's there already: https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/pull/171/files/8cfaff81885f774456cc5144ca90ced45a3fb6b1#diff-b3a9d218ef00d1a76e5c7ee743d6f30dR274

But, Actually, I want to have ingress for all the streaming job managers say like: 

`<jobname>.flink.namespace.domain.name`